src/scripts/updateTestRailTestsFromLastDay.js

```diff
@@ -14,9 +14,13 @@ const DAYS_TO_COUNT_BACK = Number(process.env.DAYS_TO_COUNT_BACK) || 1;
  */
 (() => {
   const testsToUpdate = GitUtils.getMergedTestsIdsSinceDay(DAYS_TO_COUNT_BACK);
-  console.log(`Updating ${testsToUpdate.size} tests...`);
-  console.log(testsToUpdate);
+  if (!testsToUpdate || testsToUpdate.size === 0) {
```

Comment:
in what scenario the array size can be 0?

Reply:
When files that are not tests files merged last day